src/dal/test/stories.py

```
@@ -268,21 +271,24 @@ def __init__(self, case, inline_number, inline_related_name=None,
-                    )
-                ).first  # as usual, rely on implicit wait
-            except Exception:
-                continue
+            expected_selector = self.field_selector.replace(
+                str(self.inline_number),
+                str(num),
+            )
+            deadline = time.time() + 5
+            while time.time() < deadline:
+                try:
+                    self.case.browser.find_by_css(expected_selector).first
```

Ruff (0.15.18)

[warning] Found useless attribute access. Either assign it to a variable or remove it.

(B018)